main.go

```diff
@@ -119,6 +125,19 @@ func (configs ConfigsModel) validate() error {
 	if err := input.ValidateIfPathExists(configs.ZipPath); err != nil {
 		return fmt.Errorf("Issue with ZipPath: %s", err)
```

Comment:
Error strings should not start with capital letters. Please apply this to every occurrence in the file.

Reply:
You're right, in general, but in this case the error message is the 'top-level' error and is used by the `failf` function in main. Our logging convention is to start with a capital letter in these cases and use all lowercase in the 'embedded' error messages.